templates/cf-infrastructure-bosh-lite.yml

```diff
@@ -580,6 +580,7 @@ properties:
     scim:
       users:
       - admin|admin|scim.write,scim.read,openid,cloud_controller.admin,clients.read,clients.write,doppler.firehose,routing.router_groups.read
+      - <<: (( merge ))
```

Comment:
Spiff adds an empty field when you don't add users resulting in this:
```yaml
scim:
  users:
  - admin|admin|scim.write,scim.read,openid,cloud_controller.admin,clients.read,clients.write,doppler.firehose,routing.router_groups.read
  - {}
```

This causes the following error when deploying on bosh-lite because the UAA template can't handle an empty entry in the scim.users field:
```
Error 100: Unable to render instance groups for deployment. Errors are:
   - Unable to render jobs for instance group 'uaa_z1'. Errors are:
     - Unable to render templates for job 'uaa'. Errors are:
       - Error filling in template 'uaa.yml.erb' (line 163: undefined method `+' for nil:NilClass)
```